apps/aecore/src/aec_pow_test.erl

```diff
@@ -0,0 +1,42 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2017, Aeternity Anstalt
+%%% @doc
+%%%    A library providing PoW generation and verification for testing purposes
+%%%    The solution is always 42.
+%%%
+%%% @end
+%%%-------------------------------------------------------------------
+-module(aec_pow_test).
```

Comment:
Test files should be placed in test directory. If this file is to be used "in production", then the "_test" suffix is not well chosen.

Reply:
@ThomasArts I think the meaning of this file is `pow test engine` the same meaning as `pow cyckoo engine`, or I'm just wrong.

Reply:
yeah, that was the intended meaning, i can rename it to something else though ( maybe `aec_test_pow`)

Reply:
aec_test_pow would be a good name. The test framework picks up all _test as test cases and will start running them. Even if that's not the case, the code reader will think it's a test.

Since the module name already indicated that this is a test pow, you could easily well just call it pow().